MiniApp/Classes/MiniAppLister.swift

```diff
@@ -14,4 +14,20 @@ internal class MiniAppLister {
             }
         }
     }
+
+    func fetchMiniApp(miniAppId: String, apiClient: MiniAppClient, completionHandler: @escaping (Result<MiniAppInfo, Error>) -> Void) {
+
+        apiClient.getMiniApp(miniAppId) { (result) in
+            switch result {
+            case .success(let responseData):
+                guard let decodeResponse = ResponseDecoder.decode(decodeType: Array<MiniAppInfo>.self, data: responseData.data), let miniApp = decodeResponse.first else {
```

Comment:
Array<MiniAppInfo>.self ? I think it will return Mini App info for a given App ID, not an array ?

Reply:
no, as per documentation it will return an array of one element